hack/xbuild

```diff
@@ -8,20 +8,23 @@ LDFLAGS=$4
 # target => $GOOS $GOARCH
 # see: https://golang.org/doc/install/source#environment
 TARGETS=(
+    alpine amd64
     darwin amd64
     linux amd64
-    alpine amd64
+    windows amd64
 )
 
 for ((i=0; i<"${#TARGETS[@]}"; $((i=$i+2)) )); do
   GOOS="${TARGETS[i]}"
   GOARCH="${TARGETS[i+1]}"
-  echo $DEST/$GOOS/$GOARCH/$NAME
+  OUTPUT=$DEST/$GOOS/$GOARCH/$NAME
+  if [[ $GOOS = "windows" ]]; then OUTPUT=$OUTPUT".exe"; fi
```

Comment:
this is OK, but it's generally safer to write it as follow:
OUTPUT="$DEST/$GOOS/$GOARCH/$NAME"
[[ "$GOOS" = "windows" ]] && OUTPUT="${OUTPUT}.exe"